website/content/configuration/_index.md

```diff
@@ -4,19 +4,19 @@ weight: 4
 ---
 
 MetalLB remains idle until configured. This is accomplished by
-creating and deploying a configmap into the same namespace
+creating and deploying a config map into the same namespace
```

Comment:
This is referencing a type of Kubernetes resource.  Maybe it should be `ConfigMap` ?
```suggestion
creating and deploying a `ConfigMap` into the same namespace
```

Reply:
I favor lower case text without styling whenever possible.  Overwhelmingly, references to API objects like config maps, deployments, pods, and so on are more readable as plain text.  I acknowledge that this is often subjective.

Reply:
Thanks. I was mostly interested in your opinion and happy to go with what you suggest.